client/src/lifecycleManager.js

```diff
@@ -107,10 +107,10 @@ class LifecycleManager extends LuigiClientBase {
    * @private
    * @memberof Lifecycle
    */
-  _callAllFns(objWithFns, payload) {
+  _callAllFns(objWithFns, payload, origin) {
```

Comment:
This method is being called by the `_notifyUpdate` without _origin_, which means we are calling the listener handler in this case with origin _undefined_ I guess that is fine since _undefined_ is the same as no parameter.

Reply:
yes, this is intentionally